### PR TITLE
Add pinning info for libpcap

### PR DESCRIPTION
**Pull request by nehaljwani**
https://abi-laboratory.pro/tracker/timeline/libpcap/

### CODE REVIEW

File: scripts/pin_the_slow_way.py [hunk 1]
```diff
@@ -63,6 +63,7 @@
           'libgdal': 'libgdal 2.1.*',  # 2.1.0
           'libmatio': 'libmatio 1.5.*',  # NA
           'libnetcdf': 'libnetcdf 4.4.*',  # 4.4.1
+          'libpcap': 'libpcap 1.*',  # NA
```
Comment (jakirkham) [inline scripts/pin_the_slow_way.py]: This feels a little loose compared to the ABI report. Any thoughts on making it `1.8.*`?

Comment (nehaljwani) [reply scripts/pin_the_slow_way.py]: Yes, I felt so too. I'll make it `1.8.*`.

Comment (nehaljwani) [reply scripts/pin_the_slow_way.py]: Updated.

**Comment by jakirkham** [comment]
Thanks @nehaljwani. Sorry for the delay. Fixed a merge conflict that snuck in.

**Pull request merged**